ci/release-it.yml

```
@@ -0,0 +1,33 @@
+github:
+  release: true
+  draft: true
+  preRelease: true
+  releaseName: "v${version}"
+  releaseNotes: node ./ci/scripts/fetch_changelog.js
+  assets:
+    - "*.AppImage"
+    - "*.dmg"
+    - "*.exe"
```

Maybe we want to include the .asc files as well?

```suggestion
    - "*.AppImage"
    - "*.AppImage.asc"
    - "*.dmg"
    - "*.dmg.asc"
    - "*.exe"
    - "*.exe.asc"
```

---

Accepted, great stuff! I have just written "fixup!" manually in front of the commit which should work I guess.. dbb4fd9